lib/client.js

```diff
@@ -317,6 +321,8 @@ class Client extends EventEmitter {
       return
     }
 
+    opts.requestTimeout = opts.requestTimeout || this[kRequestTimeout]
```

Comment:
I think this should be `timeout` here. `requestTimeout` is kind of double in this context, i.e.

```client.request({ timeout: 10e3, ...opts })```

Reply:
Not sure I'm following 🤔

Reply:
The option to `request` should be `timeout` instead of `requestTimeout`. Otherwise it's duplicate. In my opinion.

Reply:
I see your point!
I agree that it's a "useless" duplication, but on the other hand by keeping the same name as the global option it is more intuitive when you think about overriding the globals. What do you think?

Reply:
`request({ timeout })` is likely nicer. However there might be some confusion with the socket timeout, which has a completely different meaning.